sample_app/sampleapp_common.c

```diff
@@ -365,6 +365,10 @@ static int app_reset_ind(
          reset_mode);
    }
 
+   if (should_reset_application == true){
+         os_clear_blob(PNET_FILE_INDEX_USER);
```

Comment:
Cant we call it os_clear_file? or os_clear_info...

Reply:
Removed the user part of the storage